lib/public/components/Filters/common/filters/NumericComparisonOperatorSelectionModel.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { SelectionDropdownModel } from '../../../common/selection/dropdown/SelectionDropdownModel.js';
+
+/**
+ * Model storing state of a selection of predfined numerical comparison operators
+ */
+export class NumericComparisonOperatorSelectionModel extends SelectionDropdownModel {
```

Comment:
Why `Numeric` ? You can perfectly use this with strings
Why extending `SelectionDropdownModel` if you don't use it as dropdown?

Reply:
changed, removed `Numeric`, extend `SelectionModel`